.vscode/launch.json

```diff
@@ -0,0 +1,134 @@
+{
+    // see https://github.com/microsoft/vscode-cmake-tools/blob/main/docs/debug-launch.md
+    "version": "0.2.0",
+    "configurations": [
+        // sclang target
+        {
+            "type": "lldb",
+            "request": "attach",
+            "name": "macOS: sclang attach (lldb)",
+            "program": "${workspaceFolder}/build/lang/Debug/sclang",
+            "waitFor": true,
+        },
+        {
+            "name": "macOS: sclang launch (lldb)",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/build/lang/Debug/sclang",
+            "args": [],
+            "stopAtEntry": true,
+            "cwd": "${fileDirname}",
```

Comment:
I see some benefits but also some hazards to having the cwd be based on the current file - this could create differences in behavior depending on which filed you're viewing, which could be incredibly confusing esp for someone that doesn't know about vscode launch settings. Maybe a more stable option is `${command:cmake.getLaunchTargetDirectory}`?

Reply:
Maybe `"cwd": "${workspaceFolder}",`?